docs/packages/pkg/zlog.rst

```diff
@@ -0,0 +1,20 @@
+.. spelling::
+
+    zlog
+
+.. index:: Logging ; zlog
+
+.. _pkg.zlog:
+
+zlog
+====
+
+-  `Official <https://hardysimpson.github.io/zlog>`__
+-  `Hunterized <https://github.com/hunter-packages/zlog>`__
```

Comment:
One nit: we don't have this fork so can you remove this line? TIA

Reply:
Done !